### DISCUSSION TITLE
Disclose the timeline for updating the workshop addon

### DISCUSSION THREAD

**Discussion by TiberiumFusion** [CapsAdmin/pac3#1230] [General]
Hi,

Last week, I discovered that #1221 is preventing me from loading my pac outfits, lest pac completely shit the bed. The issue is resolved here in the master branch, but the month-old fix has yet to make it to the workshop version, which has been affected for over 2 months now. I would like to know what the timeline is for releasing workshop updates, and what the hold-up is.

Thanks.

**Comment by thegrb93** [comment]
I think it just updates whenever someone wants to. It could be automated to update monthly perhaps.

**Comment by TiberiumFusion** [comment]
> I think it just updates whenever someone wants to.

I was worried this might be the case. Not "oh surprise surprise pac actually has a large test suite and it's failing right now" or something else reasonable. I'd like to be proven wrong, though...

I know release engineering is a little much for a gmod addon, but pac is one of the most ubiquitous gmod addons in history, with 790,000 current subscribers. I don't like the idea of arbitrarily forcing updates at a regular interval, but it needs/would benefit from at least some planning. At the very least, the idea of hotfixes should exist. As a player, it is frustrating to be unable to use my outfits on public servers, but it is especially aggravating that the fix for #1221 has existed for at least 4 weeks but is sitting ignored. People are actively commenting about this bug on the workshop page.

> whenever someone wants to

I'm a "someone", so could **I** compel an update to the workshop addon? Or is this something only capsadmin and his inner circle can do?

**Reply by thegrb93** [reply]
The main issue is it has to be updated by someone willing to fix anything that the update breaks.

**Reply by thegrb93** [reply]
I don't think any of the contributors workshop contributors are currently active in gmod, myself included.

**Reply by thegrb93** [reply]
I wouldn't mind creating a windows task to update it once a month though.

**Reply by TiberiumFusion** [reply]
> The main issue is it has to be updated by someone willing to fix anything that the update breaks.

I suppose this means whoever who pushed the currently broken workshop version wasn't willing to do this.

**Reply by thegrb93** [reply]
Or the issues weren't known about for a while. I've updated it now and added an automatic update task.
